killercoda-sample/karmada-CLI-installtion-example/foreground.sh

```diff
@@ -0,0 +1,63 @@
+echo waiting for init-background-script to finish
```

Comment:
Add the following command to the header of this file(Exit immediately if an error occurs.):
```sh
set -o errexit
set -o nounset
set -o pipefail
```